arangod/Aql/Executor/ShortestPathExecutor.tpp

```diff
@@ -154,8 +161,10 @@ ShortestPathExecutor<FinderType>::ShortestPathExecutor(Fetcher&, Infos& infos)
       _pathBuilder{&_trx},
       _posInPath(0),
       _pathLength(0),
-      _sourceBuilder{},
-      _targetBuilder{} {
+      _sourceBuilder(std::make_shared<velocypack::SupervisedBuffer>(
```

Comment:
Can't the supervised buffer be built on the stack as _sourceBuilder is a class member, is gonna linger there, and no one's gonna steal the buffer from it?